app/services/articles/feeds/weighted_query_strategy.rb

```diff
@@ -576,6 +607,12 @@ def configure!(scoring_configs:)
             cases: scoring_config.fetch(:cases),
             fallback: scoring_config.fetch(:fallback),
           )
+
+          # Make sure that we consider all of the days for which we're
+          # establishing cases and for which there is a fallback.
+          if valid_method_name == :daily_decay_factor
+            @published_since = scoring_config.fetch(:cases).count + 1
```

Comment:
so I'm following along - if delay decay factor was provided as a configured scoring method, and we have N cases, ensure we're looking at N + 1 days.

Reply:
```suggestion
            @published_since = (scoring_config.fetch(:cases).count + 1).days.ago
```